wordpress-mobile/WordPress-iOS
# WPWebViewController Revamp!

#### What's new:

- Bottom Toolbar has been nuked. We now have a UIRefreshControl that handles Pull to Refresh gestures.
- Back / Forward buttons are gone. If the user attempts to open an external link, the app will fall back to Safari (prior user's approval).
- UIActivitySpinner has been replaced by UIProgressView.
- Nuked a whole lot of code!.


@aerych + @diegoreymendez may i bother you with a review?. Although this was spoken via Slack, i'd like to triple confirm if the `Open in Safari` alertView is really what we want.

Thank you!

Closes #3722

--

#### Testing Scenario A: View Site
1. Launch WPiOS
2. Tap over **My Sites**
3. Pick any of your sites
4. Tap over the **View Site** row

As a result, WPWebViewController should be onscreen.
